packages/core/workers/src/child.js

```diff
@@ -39,7 +39,6 @@ export class Child {
   loggerDisposable: IDisposable;
   child: ChildImpl;
   profiler: ?Profiler;
-  workerApi: WorkerApi;
```

Comment:
What's declared below in the function initializer isn't exactly this type, which seems problematic. Any thoughts @devongovett? I assume up until now, Flow just preferred the latter.